src/main/kotlin/util/utils.kt

```diff
@@ -239,3 +296,58 @@ inline fun <reified T> Iterable<*>.firstOfType(): T? {
 }
 
 fun libraryKind(id: String): LibraryKind = LibraryKind.findById(id) ?: LibraryKind.create(id)
+
+fun String.isJavaKeyword() = javaKeywords.contains(this)
```

Comment:
I think this can be replaced with `com.intellij.lang.java.lexer.JavaLexer#isKeyword`.

Reply:
Ahh knew there must have been something